src/codeu/chat/common/BasicView.java

```diff
@@ -52,4 +52,6 @@
   //   Return all messages whose id is found in the given collection.
   Collection<Message> getMessages(Collection<Uuid> ids);
 
+	//ServerInfo getInfo();
```

Comment:
I think we could uncomment this and add it to the BasicView, but I'm not completely sure. If we are not going to uncomment it, then it's probably better to delete the commented line